src/environments/environment.common.ts

```diff
@@ -43,6 +42,22 @@ export const environment: GlobalConfig = {
       timePerMethod: {[RestRequestMethod.PATCH]: 3} as any // time in seconds
     }
   },
+  // Authority settings
+  auth: {
+    // Authority UI settings
```

Comment:
Tiny note... these are "Authentication" settings, not "Authority" settings.  So, the two comments here should be corrected (on line 45 and 47)